responses/nl/HassTurnOff.yaml

```diff
@@ -5,10 +5,9 @@ responses:
       default: "Uitgezet"
       lights_area: Verlichting uitgezet
       fans_area: Ventilatoren uitgezet
-      cover: "Gesloten"
+      cover: "Wordt gesloten"
```

Comment:
This seems wrong. Actions return after they have been performed. Not all integrations do, but is the general way things should be handled from our end.

This change seems to be doing the opposite of how things should work.

Reply:
The action has been performed, the integration received the command to close the cover. However, it will take some time before the cover is actually physically closed. However, I reverted the change, and it's `Gesloten` and `Geopend` now.
Like you said, it was not related to the main concern for this PR.